docker-vms/vxfreeswitch/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+EXPOSE 8000
```

Comment:
The 8000 is mean to be for FreeSWITCH to connect to. Maybe I should just
change it to 8021 so it matches the default.